ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -44,7 +54,7 @@ Item {
     }
     
     function showSigningPhrasePopup(){
-        if(!hideSignPhraseModal && !RootStore.hideSignPhraseModal){
+        if(!hideSignPhraseModal && !RootStore.hideSignPhraseModal && visible && appMainVisible){
```

Comment:
we should add a check if popup is already `opened` too? so that we dont call popup again and again.